src/config.rs

```diff
@@ -126,3 +207,33 @@ fn usage(prog: &str) -> ! {
     .ok();
     process::exit(1)
 }
+
+/// Try to find a `snare.conf` file.
+fn search_snare_conf() -> Option<PathBuf> {
+    for cnd in SNARE_CONF_SEARCH {
+        if cnd.starts_with("~/") {
+            let mut homedir =
+                match get_user_by_uid(get_current_uid()).map(|x| x.home_dir().to_path_buf()) {
```

Comment:
You could use `$HOME` from the environment, and drop the dependency on `users`?

Reply:
I copied this idea from extsmaild -- I no longer remember why I did this other than I think I had some reason to do so. Maybe because things like `sudo` don't get `$HOME` correct?

Reply:
Hrm, yeah not sure what the desired outcome is, but `sudo echo $HOME` does not print `/root`.

Again I don't feel strongly.

Reply:
OK, then I think `get_user_by_uid` is the right thing to do, since it's more reliable than `$HOME`.